src/main/java/core/basesyntax/strategy/OperationsStrategy.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.OperationType;
+import core.basesyntax.strategy.impl.BalanceOperationsHandler;
+import core.basesyntax.strategy.impl.PurchaseOperationsHandler;
+import core.basesyntax.strategy.impl.ReturnOperationsHandler;
+import core.basesyntax.strategy.impl.SupplyOperationsHandler;
+
+public class OperationsStrategy {

```

Comment:
and I think you don't need this class at all. You can implement this logic in the DataProcessingService